.github/workflows/nm-benchmark.yml

```diff
@@ -138,18 +147,22 @@ jobs:
 
       - name: mount EFS
         run: |
-            sudo mkdir -m 777 -p /EFS
-            sudo mount -t nfs4 -o nfsvers=4.1,rsize=1048576,wsize=1048576,hard,timeo=600,retrans=2,noresvport ${{ secrets.HF_FS_CACHE }}:/ /EFS
-            sudo chown -R $(whoami):$(whoami) /EFS
+            if [[ ${{ inputs.label }} == *aws* ]]; then
```

Comment:
You could move this `if` to the step itself if we don't need that particular pattern matching; something like:
```
if: startsWith(inputs.label 'aws')
```

Reply:
i'll get it on the next iteration.